crates/bevy_core_pipeline/src/dof/mod.rs

```diff
@@ -889,19 +893,14 @@ impl DepthOfFieldPipelines {
 /// `sampler2DShadow` and will cheerfully generate invalid GLSL that tries to
 /// perform non-percentage-closer-filtering with such a sampler. Therefore we
 /// disable depth of field entirely on WebGL 2.
-#[cfg(target_arch = "wasm32")]
-fn depth_textures_are_supported() -> bool {
-    false
-}
-
+#[cfg(all(feature = "webgl", target_arch = "wasm32", not(feature = "webgpu")))]
+const DEPTH_TEXTURE_SAMPLING_SUPPORTED: bool = false;
 /// Returns true if multisampled depth textures are supported on this platform.
 ///
 /// In theory, Naga supports depth textures on WebGL 2. In practice, it doesn't,
 /// because of a silly bug whereby Naga assumes that all depth textures are
 /// `sampler2DShadow` and will cheerfully generate invalid GLSL that tries to
 /// perform non-percentage-closer-filtering with such a sampler. Therefore we
 /// disable depth of field entirely on WebGL 2.
-#[cfg(not(target_arch = "wasm32"))]
-fn depth_textures_are_supported() -> bool {
-    true
-}
+#[cfg(not(all(feature = "webgl", target_arch = "wasm32", not(feature = "webgpu"))))]
```

Comment:
i believe this could be shortened to `#[cfg(any(feature = "webgpu", not(target_arch = "wasm32")))]`

Reply:
Yep, that works